app/controllers/spree/admin/variants_controller.rb

```diff
@@ -4,14 +4,51 @@ module Spree
   module Admin
     class VariantsController < ResourceController
       helper 'spree/products'
+      include ProductFilterHelper
+
       belongs_to 'spree/product', find_by: :permalink
       new_action.before :new_before
 
+      def index
+        @url_filters = product_filters(request.query_parameters)
+      end
+
+      def edit
+        @url_filters = product_filters(request.query_parameters)
+
+        super
```

Comment:
same here, instead of super this is probably a one liner using render

Reply:
fixed